### PR TITLE
[Doc] Update gitian build guide to debian 8.4.0

### PR DESCRIPTION
**Pull request by fanquake**
This adds one missing screenshot, and updates two others.
It also moves the download section of the instructions below finishing the VM creation.

### CODE REVIEW

File: doc/gitian-building.md [hunk 1]
```diff
@@ -102,6 +95,13 @@ After creating the VM, we need to configure it.
 
 - Click `Ok` twice to save.
 
+Get the [Debian 8.x net installer](http://cdimage.debian.org/debian-cd/8.4.0/amd64/iso-cd/debian-8.4.0-amd64-netinst.iso) (a more recent minor version should also work, see also [Debian Network installation](https://www.debian.org/CD/netinst/)).
+This DVD image can be validated using a SHA256 hashing tool, for example on
+Unixy OSes by entering the following in a terminal:
+
+echo "7a6b418e6a4ee3ca75dda04d79ed96c9e2c33bb0c703ca7e40c6374ab4590748  debian-8.4.0-amd64-netinst.iso" | sha256sum -c
+# (must return OK)
```
Comment (MarcoFalke) [inline doc/gitian-building.md]: please make sure you also copy the formatting

**Comment by MarcoFalke** [comment]
I'd prefer to keep the old png blobs, as the new one are adding about 20 kB to the size. (The only thing that changed is the background icon?)

**Comment by MarcoFalke** [comment]
Concept ACK

**Comment by laanwj** [comment]
utACK, though I still don't like to have to update this for every minor debian version. I wish there was just a stable link.

**Pull request merged**